scripts/fetch-tags.mjs

```diff
@@ -0,0 +1,46 @@
+import { execSync } from 'child_process'
+import execa from 'execa'
+;(async () => {
+  let commitId = process.argv[process.argv.length - 1]
+
+  if (commitId.endsWith('fetch-tags.mjs')) {
```

Comment:
Is this expected?

It might make more sense to just read `process.argv[2]` or whatever param you expect

Reply:
Updated to just check for the 3rd argv item.